_episodes/08-motivation.md

```diff
@@ -36,14 +40,26 @@ of many factors:
 
 ## Teach Most Useful First
 
-People learn best when they care about a topic and believe they can master it.
-This presents us with a problem
+One element of a positive classroom environment is to find ways to motivate
```

Comment:
"One aspect of creating a positive classroom environment is to cultivate learner motivations / create a space that supports learner's motivation"

words are hard.